src/nanoemoji/reorder_glyphs.py

```diff
@@ -0,0 +1,71 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Helps with glyph reordering.
+
+See https://docs.google.com/document/d/14b8bivUBdWSkeGLIVcqVLBVWNbXXFawguq52D2olvEA/edit for context."""
+
+
+from fontTools import ttLib
+from fontTools.ttLib.tables import otTables as ot
+from nanoemoji.util import bfs_base_table, require_fully_loaded, SubTablePath
+from typing import List
+
+
+_COVERAGE_ATTR = "Coverage"  # tables that have one coverage use this name
+
+
+# (Type, Optional Format) => List of (attr name of Coverage, optional attr name of list sorted by that coverage)
+# See Cosimo's doc for context
+_REORDER_NEEDED = {
+    (ot.SinglePos, 1): [(_COVERAGE_ATTR, None)],
+    (ot.SinglePos, 2): [(_COVERAGE_ATTR, "Value")],
+    (ot.PairPos, 1): [(_COVERAGE_ATTR, "PairSet")],
+    (ot.PairPos, 2): [(_COVERAGE_ATTR, None)],
+}
+
+
+def _access_path(path: SubTablePath):
+    path_parts = []
+    for entry in path:
+        if not entry.name:
```

Comment:
it's impossible for entry to not have a name (str), object attributes must have a non-empty str name

Reply:
oh you constructed a SubTableEntry("", root) below, that's what this if is catching

Reply:
Updated to pass font["tag"].table as the first entries accessor